### PR TITLE
Hsm 432/hardcode pub gpg keys

### PR DESCRIPTION
**Pull request by islamaminBitGo**
Ticket: HSM-432

### CODE REVIEW

**Comment by pengyuc-bitgo** [comment]
Looks like some tests need to be fixed.

File: modules/bitgo/test/v2/unit/internal/tssUtils/bitgoMpcGpgPubKeys.ts [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+  public async testPickBitgoPubGpgKeyForSigning(
+    isMpcv2: boolean,
+    reqId?: IRequestTracer,
+    enterpriseId?: string
+  ): Promise<openpgp.Key> {
+    return this.pickBitgoPubGpgKeyForSigning(isMpcv2, reqId, enterpriseId);
+  }
+}
+
+describe('TSS MPC Pick BitGo GPG Pub Key Utils:', async function () {
```
Comment (OttoAllmendinger) [inline modules/bitgo/test/v2/unit/internal/tssUtils/bitgoMpcGpgPubKeys.ts:37]: `describe async` means everthing inside is a noop

I'm very surprised this isn't an eslint rule

**Review by OttoAllmendinger — CHANGES_REQUESTED** [review]

File: modules/bitgo/test/v2/unit/internal/tssUtils/bitgoMpcGpgPubKeys.ts (hunk 1 above)
Comment (islamaminBitGo) [reply modules/bitgo/test/v2/unit/internal/tssUtils/bitgoMpcGpgPubKeys.ts:37]: you're right did some reading and mocha doesnt handle a promise returned by describute

Comment (islamaminBitGo) [reply modules/bitgo/test/v2/unit/internal/tssUtils/bitgoMpcGpgPubKeys.ts:37]: great catch

Comment (islamaminBitGo) [reply modules/bitgo/test/v2/unit/internal/tssUtils/bitgoMpcGpgPubKeys.ts]: fixed

**Review by OttoAllmendinger — APPROVED** [review]

**Review by pengyuc-bitgo — APPROVED** [review]

**Review by alebusse — APPROVED** [review]

**Pull request merged**